### DISCUSSION TITLE
what's the real difference between intel and the new key?

### DISCUSSION THREAD

**Discussion by shreefentsar** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#235] [Q&A]
Hello, I know the features difference from the written in the readme, but which one is better for the battery life? because i noticed in the new key that the mac still squeeze the battery for more 1-2 percent every few hours ( it harm the cycle count ) but is this useful? I mean away from the prevent sleep and allow discharging, which one is better for the battery lifetime and cycle count?

**Comment by MatthiasKerbl** [comment]
Hi @shreefentsar,
I don't think there is much of a difference between the two modes regarding the battery life.

**Comment by shreefentsar** [comment]
which one was already been used before we have 2 options? I feel that one of them is making cycle count increase faster than the other but I can't get which one , I need to keep use the old one that was always there in the old versions

**Reply by MatthiasKerbl** [reply]
To use the original pathway you have to enable the feature "Use Classic SMC key(Intel)".
